dev/ci/docker_linux/Dockerfile

```diff
@@ -46,25 +45,29 @@ RUN apt-get update && apt-get install -y google-cloud-sdk && \
     gcloud config set core/disable_usage_reporting true && \
     gcloud config set component_manager/disable_update_check true
 
-# Add nodejs repository to apt sources and install it.
-ENV NODEJS_INSTALL="/opt/nodejs_install"
-RUN mkdir -p "${NODEJS_INSTALL}"
-RUN wget -q https://deb.nodesource.com/setup_10.x -O "${NODEJS_INSTALL}/nodejs_install.sh"
-RUN bash "${NODEJS_INSTALL}/nodejs_install.sh"
+# Add repo for OpenJDK 8 from JFrog.io
+RUN wget -q -O - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | apt-key add -
```

Comment:
By default, we should be pulling from CIPD - https://chrome-infra-packages.appspot.com/p/flutter_internal/java/openjdk/linux-amd64/+/

Reply:
OK, so what would that look like here?

Reply:
@godofredoc do you know of any existing cirrus scripts that download deps from CIPD?

Reply:
ditto here about access flutter_internal

Reply:
Downloading from the public CIPD is trivial if we download depot_tools, flutter_internal should not be used outside of LUCI.